common/repeat_timer.go

```diff
@@ -31,20 +33,29 @@ func (t *DefaultTicker) Chan() <-chan time.Time {
 	return t.t.C
 }
 
-// Implements Ticker
-func (t *DefaultTicker) Stop() {
+// Implements Ticker. Returns true if the ticker is stopped,
+// false if it was already stopped.
+func (t *DefaultTicker) Stop() bool {
+	if t.stopped {
```

Comment:
race condition

Reply:
not meant to be called concurrently. 

Locking handled by the RepeatTimer